docs/sources/operators-guide/securing/authentication-and-authorization.md

```diff
@@ -7,7 +7,9 @@ weight: 20
 
 # Grafana Mimir authentication and authorization
 
-Grafana Mimir is a multi-tenant system where tenants can query metrics and alerts that include their tenant ID. The query takes the tenant ID from the `X-Scope-OrgID` parameter that exists in the HTTP header of each request.
+Grafana Mimir is a multi-tenant system where tenants can query metrics and alerts that include their tenant ID.  
+The query takes the tenant ID from the `X-Scope-OrgID` parameter that exists in the HTTP header of each request (e.g. `X-Scope-OrgID: tenant-id`).  
+Queries can also be federated across multiple tenants when `tenant-federation` is enabled, refer to [Configuration Parameters]({{< relref "../configuring/reference-configuration-parameters/index.md#configuration-parameters" >}}). The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header (e.g. `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`).
```

Comment:
[nit] I would be more direct about how to enable tenant federation.

```suggestion
Queries can also be federated across multiple tenants when tenant federation is enabled setting `-tenant-federation.enabled=true`.
The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header (e.g. `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`).
```

Reply:
> [nit] I would be more direct about how to enable tenant federation.

```suggestion
You can federate queries across multiple tenants by using `true` in `-tenant-federation.enabled=true`. When you specify tenant IDs, separate them with a pipe (`|`) character in the 'X-Scope-OrgID' header, as in the example `X-Scope-OrgID: tenant-1|tenant-2|tenant-3`.
```